### PR TITLE
127/popout

### LINKED ISSUE #127: add pop-out window to comments section
ckeditor might have an option for this

### PR DESCRIPTION
**Pull request by acholyn**
Enables a popout window on all ckeditor instances. The new window will send the content back to the original editor when "save" is clicked and will do the same and close the window when "save and close" is clicked. There should be a descriptive title for the window and the content in case several are open.

To note: the original plan was to have it just for the comments section but the way we have the config set up and initialise the editors makes it difficult to selectively update the config

closes #127

### CODE REVIEW

File: mod_app/static/ckeditor/ckeditor/plugins/popout/plugin.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            });
+
+          newWindow.document
+            .getElementById("save-button")
+            .addEventListener("click", function () {
+              // Transfer content from the new editor to the original editor
+              editor.setData(
+                newWindow.CKEDITOR.instances["popout-editor"].getData()
+              );
+            });
```
Comment (tcouch) [inline mod_app/static/ckeditor/ckeditor/plugins/popout/plugin.js:91]: I don't understand the need for this if the content is already being synced on change (lines 62-69).

**Review by tcouch — APPROVED** [review]
Seems to work as expected, just not sure about the need for two save buttons.

File: mod_app/static/ckeditor/ckeditor/plugins/popout/plugin.js (hunk 1 above)
Comment (acholyn) [reply mod_app/static/ckeditor/ckeditor/plugins/popout/plugin.js:91]: ah whoops, I meant to delete the syncing; it only works if you're quite slow so opted for the save options

**Comment by acholyn** [comment]
> just not sure about the need for two save buttons.

I figured they're probably used to having this sort of flow with django admin + it allows for slightly different work styles. If they find they don't need one, it'll be easy to remove it as well since the new window has a very simple structure

**Pull request merged**